app/static/js/userProfile.js

```diff
@@ -151,5 +169,39 @@ function updateManagers(el, volunteer_username ){// retrieve the data of the stu
             "program_id":program_id,       // program id
             "action":action,          //action: add or remove
              },
+
+     success: function(s){
+         if(action == "add"){
+             if ($("#liveToast").is(":visible") == true){
+                 toastList[0].hide()
+                 $("#toastDiv").empty()
+                 notification.appendTo("#toastDiv")
+                 programManagerToastElements = [].slice.call(document.querySelectorAll('.toast'))
+                 programManagerToastList = programManagerToastElements.map(function (toastEl) {
+                    return new bootstrap.Toast(toastEl)
+                })
+             }
+             $("#toast-body").html(addMessage)
+             toastList[0].show()
+         }
+
+         if(action == 'remove'){
```

Comment:
Almost everything below this line is repetition, instead, you could just complete lines 175 through 181. Afterward, you could then check if the action was "add" or "Remove" and then have the flasher display accordingly.